notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Messages/{id}: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
+    get: 
+      operationId: "getMessage"
+      summary: "Get Message"
+      tags: 
+        - "Messages"
+      description: "Retrieve a message already submitted for delivery."
+      responses: 
+        200: 
+          description: "Message found."
+          schema: 
+            $ref: "#/definitions/Message"
+  /Users/{id}/Messages: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
```

Comment:
> it's a matter of id discoverability (enumerability). service provider knows the user's uuid only for users that subscribed the service itself 

I don't understand how the service provider could get the UUID of the user since from the SPID auth he only gets the fiscal number, so we need to provide a way for the service provider to map the fiscal number to a UUID, so the problem persists (the service provider can enumerate all fiscal numbers and get their associated UUIDs)  

> while the fiscal number is a public parameter and opens the possibility for the service provider to query every user message queue.

but only trusted service providers will be able to retrieve a user messages (i.e. our app backend), nobody else

unless you intend to have users authorize the service provider to access their data through an oAuth flow, bu that we decided is out of scope for now

please explain